### ISSUE TITLE
Body shapes

### ISSUE THREAD

**Issue by ScarXLXX** [iblobtouch/iblobtouch.github.io#22]
This is not really an issue but just a suggestion.

I was trying to remake the Guardian of the Pentagons but it turns out the Triangle faces the wrong way and I can't re-arrange it.

Maybe some way to re-arrange where the tanks' body is facing?

**Comment by Leviterminator** [comment]
Yeah, but not only the triangle, the drones are facing the wrong way. You can check it, it looks a bit dumb.

**Comment by Acarii** [comment]
This is tentative, do not take it as confirmation of anything.

We discussed adding new shapes. Long standing argument about adding a stalker barrel. The idea came up to allow users to create their own shapes using something like [[0,0],[0,50],[50,0]] which would define each point of _any_ given shape you could imagine, up to the point of crashing your browser if you added enough(which is a lot of lines, don't worry). This same idea could be applied to body shapes.  

I could _easily_ add shapes for you. Any sort you could need. I could also support a rotate function. However, two things sit in my way. The first, is the above idea/pseudo-plan. It's just awesome, and relieves so many headaches for us in the long run.

The second is backwards compatibility/supporting it in export. That specific part of the code I don't understand _at all_. Its a mess, and my additions(via likcoras' help) to add colour support have already made it worse. I'm hoping that the rewrite will make things easier for me to add things like that.

**Comment by ArenaCloser** [comment]
gud idea
